scripts/build_docs.sh

```diff
@@ -97,6 +97,18 @@ else
     --source "${TMP_BUILD_FOLDER}" --destination "${OUTPUT}"
 fi
 
+if [ ! -f $OUTPUT/index.html ]; then
+    echo
```

Comment:
Elsewhere in this file we've got `echo ""` instead of a zero arg echo.  I don't think the difference matters, but maybe it's worth keeping internally consistent?  Guess we probably don't have enough bash kicking around to have a style guide for that.